fe/fe-core/src/main/java/com/starrocks/analysis/ArithmeticExpr.java

```diff
@@ -443,7 +443,7 @@ public void analyzeImpl(Analyzer analyzer) throws AnalysisException {
 
     @Override
     public int hashCode() {
-        return 31 * super.hashCode() + Objects.hashCode(op);
```

Comment:
Do you known why using 31 in the previous formula?

Reply:
I guess it's from Eclipse generate

Reply:
It seems copy from the logical in jdk.